### ISSUE TITLE
Document differences between production and local development

### ISSUE THREAD

**Issue by scottbarnes** [internetarchive/openlibrary#8687]
### Describe the problem that you'd like solved

For new contributors, a short list of the differences between the production site and the local development environment might be good, so they can better tell if their local development environment is behaving correctly.

For example, certain links (e.g. `/about`) by default won't have content locally, and the carousels on the home page look different to the production environment. This can be confusing for new contributors.

### Proposal & Constraints

Perhaps a new page on the wiki with the differences could be created, and then it could be link to from [Start the app](https://github.com/internetarchive/openlibrary/tree/master/docker#start-the-app) in the Docker documentation, and in [Logging in](https://github.com/internetarchive/openlibrary/wiki/Getting-Started#logging-in) in the Getting Started section of the Wiki.

### Stakeholders

@liaozhuzhu

**Comment by liaozhuzhu** [comment]
Something that could be helpful too for newcomers is updating the onboarding videos (e.g https://archive.org/details/openlibrary-developer-docs/openlibrary-docker-set-up.mp4). A walkthrough using [loom](https://www.loom.com/) or something to get timestamps would be nice, as well as a complete walk through from forking the repo, to making changes within the dev codebase would be cool and would cover much of what the wiki pages strive to accomplish.

**Comment by dhruvchandak30** [comment]
@liaozhuzhu  Can we add timestamps to the existing video for now until a new video is made
Similar to this?
![image](https://github.com/internetarchive/openlibrary/assets/116879117/7b19cd6f-186a-4319-92ba-b1de4e28a53d)

**Comment by liaozhuzhu** [comment]
@dhruvchandak30 yeah that'd be really nice. I was thinking that and also the way youtube does it is it provides a transcript along with timestamps of when each thing was said. The next step would then be to have different sections for example "Setting up Git _3:20_" which Loom does automatically using AI, but like you said that requires a whole new video(s) to be made.

**Comment by dhruvchandak30** [comment]
@liaozhuzhu  Can you please assign me, I would really like to work on this issue

**Comment by liaozhuzhu** [comment]
@dhruvchandak30 I can't unfortunately since I'm just a new contributor myself.

I would reach out to @scottbarnes to see if you can; however, I think this issue's main focus would be the wiki doc pages and then second priority would be the videos. I also think this issue is reserved more for higher ups if we're gonna be tampering with the wiki pages, but I'm sure Scott has more say in that.

**Comment by dhruvchandak30** [comment]
Okayy, Can I get the Slack Channel invite, so that we can discuss about both the issues in details, and I will start preparing the new page.

**Comment by scottbarnes** [comment]
@dhruvchandak30, I admire your enthusiasm. I am hopeful that you can hold on to that enthusiasm until after this issue is triaged on Monday. I would hate for you to start work and then have the requirements change, or for it to turn out the work is mostly or completely done already. I know this is a fairly straightforward issue, but maybe some or all of the work has already been done somewhere and I don't know about it, etc.

Also, have you filled out the form at https://openlibrary.org/volunteer? That may be the fastest way to getting a Slack invite.
